### ISSUE TITLE
Severity warning is being reported as error

### ISSUE THREAD

**Issue by amimas** [textlint/textlint#583]
**What version of textlint are you using?**

v11.2.1

**What file type (Markdown, plain text, etc.) are you using?**
Markdown

**What did you do? Please include the actual source code causing the issue.**
Followed the instructions from the doc here:
https://textlint.github.io/docs/configuring.html#severity-config-of-rules

Configured `.textlintrc` for one of the rule's severity to be *warning* level instead of default *error*. Here's the copy of my `.textlintrc`
```json
{
  "rules": {
    "@textlint-rule/no-unmatched-pair": true,
    "apostrophe": true,
    "en-capitalization": true,
    "max-comma": {
      "max" : 3
    },
    "no-dead-link": {
      "checkRelative": true,
      "baseURI": null,
      "ignore": [],
      "preferGET": [],
      "ignoreRedirects": true
    },
    "no-empty-section": true,
    "textlint-rule-common-misspellings": true,
    "textlint-rule-write-good": {
      "severity": "warning" 
    }
  }
}
```

**What did you expect to happen?**
Expected to see problems found by **write-good** rule being reported as "warning"

**What actually happened? Please include the actual, raw output from textlint.**
**write-good** rule related issues are being reported as "error"

debug log:
```
  textlint:cli Running on files, stdin-filename: undefined +0ms
  textlint:module-loader config Config {
  textlint:module-loader   configFile:
  textlint:module-loader    '/Users/amimas/myproject/website/.textlintrc',
  textlint:module-loader   rulesBaseDirectory: undefined,
  textlint:module-loader   rules:
  textlint:module-loader    [ '@textlint-rule/no-unmatched-pair',
  textlint:module-loader      'apostrophe',
  textlint:module-loader      'en-capitalization',
  textlint:module-loader      'max-comma',
  textlint:module-loader      'no-dead-link',
  textlint:module-loader      'no-empty-section',
  textlint:module-loader      'textlint-rule-common-misspellings',
  textlint:module-loader      'textlint-rule-write-good' ],
  textlint:module-loader   disabledRules: [],
  textlint:module-loader   filterRules: [],
  textlint:module-loader   disabledFilterRules: [],
  textlint:module-loader   presets: [],
  textlint:module-loader   plugins: [],
  textlint:module-loader   pluginsConfig: {},
  textlint:module-loader   rulesConfig:
  textlint:module-loader    { '@textlint-rule/no-unmatched-pair': true,
  textlint:module-loader      apostrophe: true,
  textlint:module-loader      'en-capitalization': true,
  textlint:module-loader      'max-comma': { max: 3 },
  textlint:module-loader      'no-dead-link':
  textlint:module-loader       { checkRelative: true,
  textlint:module-loader         baseURI: null,
  textlint:module-loader         ignore: [],
  textlint:module-loader         preferGET: [],
  textlint:module-loader         ignoreRedirects: true },
  textlint:module-loader      'no-empty-section': true,
  textlint:module-loader      'textlint-rule-common-misspellings': true,
  textlint:module-loader      'textlint-rule-write-good': { severity: 'warning' } },
  textlint:module-loader   filterRulesConfig: {},
  textlint:module-loader   rulePaths: [],
  textlint:module-loader   formatterName: undefined,
  textlint:module-loader   quiet: false,
  textlint:module-loader   color: true,
  textlint:module-loader   cache: false,
  textlint:module-loader   cacheLocation:
  textlint:module-loader    '/Users/amimas/myproject/website/.textlintcache' } +0ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/@textlint-rule/textlint-rule-no-unmatched-pair/lib/textlint-rule-no-unmatched-pair.js +4ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-apostrophe/index.js +21ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-en-capitalization/lib/textlint-rule-en-capitalization.js +0ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-max-comma/lib/textlint-rule-max-comma.js +370ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-no-dead-link/lib/no-dead-link.js +3ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-no-empty-section/lib/textlint-rule-no-empty-section.js +37ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-common-misspellings/lib/common-misspellings.js +2ms
  textlint:module-loader Loading rules from /Users/amimas/myproject/website/node_modules/textlint-rule-write-good/lib/write-good.js +4ms
  textlint:engine-core Process files [ '/Users/amimas/myproject/docs/index.md' ] +0ms
  textlint:engine-core Not Process files [] +0ms
  textlint:TextLintCoreTask rules TextlintRuleDescriptors {
  ruleDescriptorList:
   [ TextlintLintableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintFixableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintFixableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintLintableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintFixableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintLintableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintFixableRuleDescriptor { textlintKernelRule: [Object] },
     TextlintFixableRuleDescriptor { textlintKernelRule: [Object] } ] } +0ms
  textlint:TextLintCoreTask filterRules TextlintFilterRuleDescriptors { ruleDescriptorList: [] } +2ms
  textlint:core-task common-misspellings pushReport {"index":35,"fix":{"range":[35,44],"text":"absolutely","isAbsolute":false}} +0ms
  textlint:core-task max-comma pushReport {"line":0,"column":184} +8ms
  textlint:core-task max-comma pushReport {"line":0,"column":268} +1ms
  textlint:core-task write-good pushReport {"index":27} +1ms
  textlint:core-task write-good pushReport {"index":111} +0ms
  textlint:core-task write-good pushReport {"index":126} +0ms
  textlint:core-task write-good pushReport {"index":290} +0ms
  textlint:core-task write-good pushReport {"index":15} +6ms
  textlint:core-task write-good pushReport {"index":195} +1ms
  textlint:core-task apostrophe pushReport {"index":7,"fix":{"range":[7,10],"text":"js’","isAbsolute":false}} +0ms
  textlint:core-task write-good pushReport {"index":18} +8ms
  textlint:core-task write-good pushReport {"index":22} +1ms
  textlint:core-task write-good pushReport {"index":72} +0ms
  textlint:core-task max-comma pushReport {"line":0,"column":0} +6ms
  textlint:core-task apostrophe pushReport {"index":381,"fix":{"range":[381,385],"text":"it’s","isAbsolute":false}} +0ms
  textlint:core-task write-good pushReport {"index":136} +0ms
  textlint:core-task write-good pushReport {"index":363} +0ms
  textlint:core-task write-good pushReport {"index":49} +2ms
  textlint:core-task write-good pushReport {"index":35} +3ms
  textlint:core-task write-good pushReport {"index":87} +0ms
  textlint:core-task write-good pushReport {"index":36} +6ms
  textlint:core-task write-good pushReport {"index":94} +2ms
  textlint:core-task write-good pushReport {"index":130} +0ms
  textlint:core-task write-good pushReport {"index":142} +0ms
  textlint:core-task write-good pushReport {"index":164} +0ms
  textlint:core-task write-good pushReport {"index":49} +8ms
  textlint:core-task write-good pushReport {"index":44} +9ms
  textlint:core-task write-good pushReport {"index":87} +0ms
  textlint:core-task write-good pushReport {"index":67} +5ms
  textlint:core-task write-good pushReport {"index":98} +3ms
  textlint:core-task write-good pushReport {"index":122} +6ms
  textlint:core-task write-good pushReport {"index":28} +1ms
  textlint:core-task write-good pushReport {"index":109} +3ms
  textlint:core-task write-good pushReport {"index":183} +0ms
  textlint:core-task write-good pushReport {"index":222} +0ms
  textlint:core-task write-good pushReport {"index":158} +4ms
  textlint:core-task write-good pushReport {"index":167} +0ms
  textlint:core-task write-good pushReport {"index":37} +3ms
  textlint:core-task write-good pushReport {"index":33} +2ms
  textlint:core-task write-good pushReport {"index":52} +0ms
  textlint:core-task write-good pushReport {"index":146} +0ms
  textlint:@textlint/linter-formatter try formatterName: stylish +0ms

/Users/amimas/myproject/docs/index.md
   9:36   ✓ error  This is a commonly misspelled word. Correct it to absolutely    common-misspellings
  11:28   error    "is composed" may be passive voice                              write-good
  11:112  error    "is composed" may be passive voice                              write-good
  11:127  error    "various" is a weasel word                                      write-good
```

**Comment by azu** [comment]
Thanks for report!

> ```json
> 
>     "textlint-rule-common-misspellings": true,
>     "textlint-rule-write-good": {
>       "severity": "warning" 
>     }
> ```

Ah, I figure out.

These(`textlint-rule-<name>`) option values will be ignored.
Currently, It will just enabled these rule.

Workaround: you can fix it these.

```diff
-    "textlint-rule-common-misspellings": true,
+    "common-misspellings": true,
-    "textlint-rule-write-good": {
+    "write-good": {
      "severity": "warning" 
    }
```

It is a bug about [Rule name](https://textlint.github.io/docs/configuring.html#rule-name) and rule config handling.

We need to add normalization for ruleConfig.

https://github.com/textlint/textlint/blob/152140bc81c6d16b48a708902837d95b4997b925/packages/textlint/src/config/config.ts#L339-L340

**Comment by amimas** [comment]
Thanks @azu for the quick response. I'll try it soon.

**Comment by amimas** [comment]
@azu - Just writing that confirm that the suggestion above worked nicely. Now I'm getting "write-good" related issues being reported as warning.

What is the convention when using textlint? For example: following your suggestion, I also changed ` "@textlint-rule/no-unmatched-pair": true` to `no-unmatched-pair": true`, but that returned following error:

```
✖ Error
Failed to load textlint's rule module: "no-unmatched-pair" is not found.
See FAQ: https://github.com/textlint/textlint/blob/master/docs/faq/failed-to-load-textlints-module.md


✖ Stack trace
ReferenceError: Failed to load textlint's rule module: "no-unmatched-pair" is not found.
See FAQ: https://github.com/textlint/textlint/blob/master/docs/faq/failed-to-load-textlints-module.md

    at TextLintModuleResolver.resolveRulePackageName (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/engine/textlint-module-resolver.js:99:19)
    at TextLintModuleLoader.loadRule (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/engine/textlint-module-loader.js:188:43)
    at /Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/engine/textlint-module-loader.js:76:23
    at Array.forEach (<anonymous>)
    at TextLintModuleLoader.loadFromConfig (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/engine/textlint-module-loader.js:75:26)
    at TextLintEngine.AbstractTextLintEngine (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/engine/textlint-engine-core.js:97:27)
    at new TextLintEngine (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/textlint-engine.js:25:47)
    at Object.executeWithOptions (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/cli.js:136:26)
    at Object.execute (/Users/amimas/myproject/website/node_modules/textlint/lib/textlint/src/cli.js:93:25)
    at /Users/amimas/myproject/website/node_modules/textlint/bin/textlint.js:34:20
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:282:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

It'd be really helpful to have a single convention for all rules.

**Comment by azu** [comment]
> `"@textlint-rule/no-unmatched-pair": true` to `no-unmatched-pair": true`,

`@textlint-rule` is just organization name.
It is not related with conversation.

- `@org/no-unmatched-pair` -> `@org/textlint-rule-no-unmatched-pair` 
- `no-unmatched-pair`  -> `textlint-rule-no-unmatched-pair`

**Comment by amimas** [comment]
Thanks for the clarification

**Comment by azu** [comment]
We have fixed this issue in https://github.com/textlint/textlint/releases/tag/textlint%4011.2.2
